### PR TITLE
Added average ratings queries to PostReview.js

### PR DESCRIPTION
**Pull request by thaowii**
- Added an averageQuery that calculates the average of all the ratings from the reviews for the course that was just reviewed
- Added an updateQuery that updates the average ratings in the course table for the respective course

Testing:
- None yet ... waiting for Jeremy's approval of my SQL queries because I am scared to do anything

### CODE REVIEW

File: app/api/PostReview.js [hunk 1]
```diff
@@ -9,12 +9,50 @@ export default async (req, res) => {
...
       console.error('Error executing MySQL query:', err);
       res.status(500).json({ error: 'Internal Server Error' });
     } else {
       res.json({ success: true, message: 'Data inserted successfully' });
     }
   });
+
+  // SQL query to calculate the average ratings from all the reviews for the course
+  const avgQuery = `
+    SELECT
```
Comment (jeremynguyen593) [inline app/api/PostReview.js:23]: I think the query is this:

SELECT 
    AVG(Reviews.rating_one) AS avg_rating_one,
    AVG(Reviews.rating_two) AS avg_rating_two,
    AVG(Reviews.rating_three) AS avg_rating_three
FROM
    Reviews
JOIN
    Courses ON Reviews.class_id = Courses.class_id
WHERE
    Courses.number = ?

Also, you can just test this yourself and see if it updates the database to check. I wanted to check if it was right before merging, you should test it if u can (definitely test this query cuz I'm not 100% sure)

File: app/api/PostReview.js [hunk 2]
```diff
@@ -9,12 +9,50 @@ export default async (req, res) => {
...
+    console.error('Error executing AVG query:', avgError);
+    res.status(500).json({ error: 'Error calculating averages' });
+    return; // Exit the function early to avoid sending the success response
+  }
+
+  // Extract the average rating_one value from the result
+  const { avg_rating_one, avg_rating_two, avg_rating_three } = avgResult[0];
+
+  // SQL query to update the ratings for the course
+  const updateQuery = 'UPDATE courses SET rating_one = ?, rating_two = ?, rating_three = ? WHERE class_id = ?';
```
Comment (jeremynguyen593) [inline app/api/PostReview.js:46]: class_id does not correspond to class number. I think it should class_number = ?, not class_id. Also test this to make sure

**Review by jeremynguyen593 — CHANGES_REQUESTED** [review]

File: app/src/Rating/Rating.tsx [hunk 3]
```diff
@@ -85,7 +85,8 @@ export const ReviewHolder: React.FC<{classNum : string}> = ({classNum}) => {
                 },
                 body: JSON.stringify(ratingContents),
             });
-            await response.json();
+            const data = await response.json();
+            console.log(data);
```
Comment (jeremynguyen593) [inline app/src/Rating/Rating.tsx:89]: Remove console.log

**Review by jeremynguyen593 — CHANGES_REQUESTED** [review]

**Review by jeremynguyen593 — APPROVED** [review]

File: app/api/PostReview.js (hunk 2 above)
Comment (thaowii) [reply app/api/PostReview.js]: Fixed.

**Pull request merged**